src/AutoRouteManager.php

```diff
@@ -124,18 +124,41 @@ public function handleDefunctRoutes()
     }
 
     /**
+     * Find an existing route which matches the URI of the given context.
+     *
+     * It is searched within the currently processed collection and the already
+     * persisted routes (using the adapter).
+     */
+    private function findExistingRoute(UriContext $uriContext)
+    {
+        $uri = $uriContext->getUri();
+        $existingRoute = null;
```

Comment:
this line is unneccessary, as the variable is overwritten on the next code line below